source/index.html.md

```diff
@@ -50,17 +43,99 @@ $inspetor = new InspetorClient($inspetor_config);
 ?>
 ```
 
-To setup an Inspetor Library in any language you first need to set some configuration. 
+The first step of integrating Inspetor's antifraud services into your product is instantiating the library.
+
+In order to do so, you will need to pass the `InspetorClient` your product's **App ID** and **Tracker Name**, both of which will be provided to you by the Inspetor Team.
+
+_Et voilà_! You now have an Inspetor client object that is capable of sending all of the information necessary to teach Inspetor's decision layer how to prevent fraud at your company. However, there is obviously a difference between instantiating a client instance and using it properly.
+
+# Using the Client Library
+
+Inspetor prevents chargebacks due to fraudulent purposes. In order to protect your company from fraudsters, our decision models need to be informed with customer behavior.
+
+Events like logins, profile updates, or even unrelated purchases on the same account all contribute to our assessment of whether or not a given transaction might be fraudulent. But we can't do it alone--we need _you_, our client, to send this information to us via the Client Library so that we can make better-informed decisions.
 
-You will need to pass to the library your **"AppId"** and a **"TrackerName"**, both will be provided to you by the Inspetor Team.
+At a high level, our decision model understands the e-commerce world in the following primary terms:
+
+- <a href="#account">**Accounts**</a>
+- <a href="#event">**Events**</a>
+- <a href="#sale">**Sales**</a>
+- <a href="#item">**Sale Items**</a>
+- <a href="#transfer">**Transfers**</a>
+
+You can think of the relationship between those entities something like this: If a user purchases tickets for a show on your site, Inspetor interprets the action as:
+
+- the creation of a new <a href="#sale">*Sale*</a>
+- the association of that Sale with an existing <a href="#account">*Account*</a>
+- the association of that Sale with an <a href="#item">*Item*</a>
+- the association of that Item with an existing <a href="#item">*Event*</a>
+
+## When to send events to Inspetor
+
+The primary Inspetor entities are **stateful** objects. They have properites that can be updated, and the values of these properties are crucial to our evaluation of transaction validity. However, if we base our evaluation upon outdated or incorrected information, the accuracy of our evaluation is likely to suffer as well. As such, it is critical that you notify Inspetor of any changes to these properties whenever they occur via the Inspetor Client Library.
```

Comment:
**critical** (add highlight)

Reply:
I have two pretty aggressively highlighted warnings below that do the same